### ISSUE TITLE
Glossary links are missing from Success Criteria in the QuickRef document

### ISSUE THREAD

**Issue by jfhector** [w3c/wcag#762]
## The problem

The phrasing of Success Criteria often use technical terms defined in the Glossary section.

In the [main WCAG document](https://www.w3.org/TR/WCAG21/), these technical terms are links. This is helpful because:
* It clarifies that the term is a technical term with a specific meaning
* It makes it easy to look up the definition of the term

In the [QuickRef](https://www.w3.org/WAI/WCAG21/quickref/) version of the WCAG, however, technical terms in the Success Criteria are not links, and are not identified as technical terms in any way.

This leads to comprehension problems.

## A concrete example

In the [QuickRef document](https://www.w3.org/WAI/WCAG21/quickref/), the 'large text' exception to SC 1.4.3 reads like this:

"The visual presentation of text and images of text has a contrast ratio of at least 4.5:1, except for the following:
* Large Text: Large-scale text and images of large-scale text have a contrast ratio of at least 3:1;
* ... "

The term 'Large text' has a very specific meaning defined in the glossary. But in SC 1.4.3 in the QuickRef document, it is not identified as a technical term. This suggests that what constitutes 'Large text' is open to interpretation.

My colleagues' and my own failed attempts at understanding this Success Criteria have come in part from the fact that 'Large text' wasn't identified as a technical term.

This has also been my experience for all Success Criteria that mention technical terms, not just SC 1.4.3.

I love the QuickRef document. So I'm keen for others to use it without getting confused.

**Comment by alastc** [comment]
There is a particular process that sends data from the main spec to the QuickRef, @michael-n-cooper is it possible to embed glossary links into that data?

**Comment by awkawk** [comment]
Adding @yatil for this.

**Comment by yatil** [comment]
When we implemented the new WCAG Quickref, we wanted to make it quick and skimmable.

If I recall the discussion correctly, the understanding link provides a way to get to the background of an SC, including all the definitions and in-detail examples. The nature of the quickref is to be a quick way to get into WCAG, not to be a comprehensive mirror of the specification. 

Adding them now would probably mean to re-think most of the data structure, it would also have an impact on the UI. 

I’m not opposed of talking about that but as the Quickref as it is is mainly in “maintenance mode” (content updates & bug fixes), we are probably not able to act on it quickly (unless someone else can contribute their time 😉). 

(Duplicate: https://github.com/w3c/wai-wcag-quickref/issues/147)

[Administrivia: We can move this issue to the w3c/wai-wcag-quickref repository if you rather like that the issue is discussed over there, @alastc, @awkawk ]

**Comment by awkawk** [comment]
Happy to have the issue discussed in the quickref repo! :) Sounds like this was discussed and decided to be out of scope for the time being.
